official/vision/beta/projects/deepmac_maskrcnn/serving/detection.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Thanks! Can you please add a unittest for this file? you can find similar one here: https://github.com/tensorflow/models/blob/master/official/vision/beta/serving/detection_test.py

Reply:
Unit test file added.